src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -23,7 +25,7 @@
             + "Parameters:\n1. name (must be a valid name, not case-sensitive)\n"
             + "2. index (must be a valid index, no negative numbers)\n\n"
             + "Example: " + COMMAND_WORD + " aglio olio\n"
-            + "Example: " + COMMAND_WORD + " -x 1";
+            + "Example: " + COMMAND_WORD + " " + PREFIX_INDEX + "1";
```

Comment:
Good catch seeing the hardcoding and potential problems this might cause if `PREFIX_INDEX` is modified in the future! 👍